### DISCUSSION TITLE
windows + windows.h (evil macros) -> Canvas.DrawText not working

### DISCUSSION THREAD

**Discussion by SColibri** [ArthurSonzogni/FTXUI#391] [General]
# Hi!

I'm working with "windows.h" for dynamic library linking in my project. This header file includes the "winuser.h" and this one contains a macro named DrawText. This changes the definition of the function DrawText inside canvas.

Maybe not an issue with FTXUI library itself, but might help to know that this problem exists when including windows.h

My solution to this problem, I changed the function definition (maybe there is a better way?).

**Comment by ArthurSonzogni** [comment]
Fixed by @SColibri patch, by undef the Microsoft's macro.
